docs/interfaces.md

```diff
@@ -45,6 +45,14 @@ apps interfering with one another.
 Usage: reserved
 Auto-Connect: yes
 
+### pulseaudio
+
+Can access the PulseAudio sound server. Allows for sound playback in games and
+media application.
+
+Usage: common
+Auto-Connect: yes
```

Comment:
I think this is reasonable. We should perhaps consider SRU-ing the trust store as well so that applications like firefox can use this for media recording (hangouts, certain type of games, etc).